core/store.go

```diff
@@ -181,6 +181,9 @@ func DelByPtr(ptr unsafe.Pointer) bool {
 		delete(expires, obj)
 		delete(keypool, *((*string)(ptr)))
 		KeyspaceStat[0]["keys"]--
+
+		key := *((*string)(ptr))
+		WatchChannel <- WatchEvent{key, "DEL", obj}
```

Comment:
Can you please add test case also for this change?

Reply:
We would skip the test case for now. @soumya-codes would create a separate issue for store.